clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -113,7 +113,8 @@ New check aliases
 Changes in existing checks
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- Added `ctime` and `localtime` to clang-tidy.
+- New unsafe functions checks :doc:`bugprone-unsafe-functions-check`
```

Comment:
Format is still inconsistent with other entries.

Reply:
I think I understand now. I had the wrong path. I have fixed it now.

Reply:
The `:doc:` was also wrong. I have fix that now.